lib/kubernetes-deploy/resource_watcher.rb

```diff
@@ -41,14 +41,19 @@ def run(delay_sync: 3.seconds, reminder_interval: 30.seconds)
     def report_what_just_happened(resources, watch_time)
       new_successes, new_failures = resources.partition(&:deploy_succeeded?)
       new_failures.each do |resource|
+        tags = %W(namespace:#{@namespace}, sha:#{ENV['REVISION']}, resource:#{resource.id})
```

Comment:
eg. ```[StatsD] measure kubernetes_deploy.duration.success:2200.0 #namespace: #sha:6795b97b6423a33d571c2359efa8399f4e6fa0de #resources:ConfigMap/kube-shellService/memcachedDeployment/memcachedIngress/nginx```

Though I'm somewhat pondering if this should be dropped altogether since due to the interval of checks whats going on we are not really super correct about duration